src/main/java/gregtech/api/unification/material/materials/OrganicChemistryMaterials.java

```diff
@@ -0,0 +1,466 @@
+                .build();
+
+        SaltWater = new Material.Builder(1053, "salt_water")
+                .fluid()
+                .color(0x0000C8)
+                .components(Salt, 1, Water, 1)
+                .build();
+
+        Ethanol = new Material.Builder(1054, "ethanol")
+                .fluid()
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 2, Hydrogen, 6, Oxygen, 1)
+                .build();
+
+        Toluene = new Material.Builder(1055, "toluene")
+                .fluid()
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 7, Hydrogen, 8)
+                .build();
+
+        DiphenylIsophtalate = new Material.Builder(1056, "diphenyl_isophthalate")
+                .fluid()
+                .color(0x246E57)
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 20, Hydrogen, 14, Oxygen, 4)
+                .build();
+
+        PhthalicAcid = new Material.Builder(1057, "phthalic_acid")
+                .fluid(Material.FluidType.FLUID, true)
+                .color(0xD1D1D1)
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 8, Hydrogen, 6, Oxygen, 4)
+                .build();
+
+        Dimethylbenzene = new Material.Builder(1058, "dimethylbenzene")
+                .fluid()
+                .color(0x669C40)
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 8, Hydrogen, 10)
+                .build()
+                .setFormula("C6H4(CH3)2", true);
+
+        Diaminobenzidine = new Material.Builder(1059, "diaminobenzidine")
+                .fluid()
+                .color(0x337D59)
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 12, Hydrogen, 14, Nitrogen, 4)
+                .build();
+
+        Dichlorobenzidine = new Material.Builder(1060, "dichlorobenzidine")
+                .fluid()
+                .color(0xA1DEA6)
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 12, Hydrogen, 10, Chlorine, 2, Nitrogen, 4)
+                .build();
+
+        Nitrochlorobenzene = new Material.Builder(1061, "nitrochlorobenzene")
+                .fluid()
+                .color(0x8FB51A)
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 6, Hydrogen, 4, Chlorine, 1, Nitrogen, 1, Oxygen, 2)
+                .build();
+
+        Chlorobenzene = new Material.Builder(1062, "chlorobenzene")
+                .fluid()
+                .color(0x326A3E)
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 6, Hydrogen, 5, Chlorine, 1)
+                .build();
+
+        Octane = new Material.Builder(1063, "octane")
+                .fluid()
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 8, Hydrogen, 18)
+                .build();
+
+        EthylTertButylEther = new Material.Builder(1064, "ethyl_tertbutyl_ether")
```

Comment:
The correct name for this material is "Ethyl Tert-Butyl Ether." It's sort of ambiguous whether we turn the "-" into an underscore or remove it entirely for unlocalized names. We should probably be consistent in this regard, so I would compare this with Nionbium-Titanium, as well as Tungstensteel, since they each handle two word material names differently.